crates/ra_ide/src/completion/completion_context.rs

```diff
@@ -53,6 +53,8 @@ pub(crate) struct CompletionContext<'a> {
     pub(super) after_if: bool,
     /// `true` if we are a statement or a last expr in the block.
     pub(super) can_be_stmt: bool,
+    /// `true` if we expect an expression at the cursor position.
+    pub(super) can_be_expr: bool,
```

Comment:
wouldn't a more fitting name be `is_expr` here? :smile: `can_be_stmt` is named that way because it could also be an expr, isn't it?

Reply:
something something foolish consistency is the hobgoblin of little minds :)

#5297